### DISCUSSION TITLE
How do i reverse windows defender antivirus disable it always shows active

### DISCUSSION THREAD

**Discussion by 10ANT** [Greedeks/GTweak#4] [Problematic issues]
![image](https://github.com/user-attachments/assets/b768922e-869e-4a54-9922-4cde2006a891)

**Comment by Greedeks** [comment]
While staying on the "system" page, use the tweak. Wait for the reboot notification, after which the tweak will become available for Windows Antivirus to be turned on to its original state. I am aware of the problem, I am reworking this tweak, and an update will be released in the near future. Thank you for your understanding and patience! 💓 
<a href="https://github.com/Greedeks/GTweak/releases/tag/4.90.4">Solved</a>
